test/Microsoft.AspNetCore.Server.IISIntegration.Tests/IISMiddlewareTests.cs

```diff
@@ -150,15 +156,62 @@ public void PathBaseHiddenFromServer()
                 .UseSetting("PORT", "12345")
                 .UseSetting("APPL_PATH", "/")
                 .UseIISIntegration()
+                .ConfigureServices(services => services.AddAuthentication())
```

Comment:
make a copy of this test without AddAuthentication and test auth.GetSchemeAsync("Windows"); returns null.